src/algebra/classical_lie_algebras.lean

```diff
@@ -84,7 +84,7 @@ begin
   let B := Eb R j i hij.symm,
   intros c,
   have c' : A.val ⬝ B.val = B.val ⬝ A.val := by { rw [←sub_eq_zero, ←sl_bracket, c.abelian], refl, },
-  have : 1 = 0 := by simpa [matrix.mul_val, hij] using (congr_fun (congr_fun c' i) i),
+  have : (1 : R) = 0 := by simpa [matrix.mul_val, hij] using (congr_fun (congr_fun c' i) i),
```

Comment:
This is an unrelated proposed change due to an observation of @kckennylau https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/a.20puzzle